dotnet/src/webdriver/ICommandExecutor.cs

```diff
@@ -39,5 +40,13 @@ public interface ICommandExecutor : IDisposable
         /// <param name="commandToExecute">The command you wish to execute</param>
         /// <returns>A response from the browser</returns>
         Response Execute(Command commandToExecute);
+
+
+        /// <summary>
+        /// Executes a command Asynchronously
```

Comment:
```suggestion
        /// Executes a command asynchronously.
```

The capital letter fix. Additionally, I would recommend to end all sentences in XML docs (`<summary>`, `<param>`, etc.) with a period.